### DISCUSSION TITLE
Welcome to Massdriver Discussions!

### DISCUSSION THREAD

**Discussion by coryodaniel** [massdriver-cloud/docs#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tmoore78** [comment]
Just getting started using Massdriver. Looking forward to learning how it can improve my teams efficiency and enable them to focus more on delivery than the mechanisms that enable them to do so efficiently.
